### DISCUSSION TITLE
Downloaded world not generating new chunks

### DISCUSSION THREAD

**Discussion by zombiejustice** [mircokroon/minecraft-world-downloader#461] [Q&A]
When I try to play the downloaded world, it warns me that the world is experimental. I can walk all around on downloaded chunks with no issue, but the world will not generate new chunks. Any way to fix this, or any workarounds?

**Comment by zekeflynn** [comment]
Yeah I've been trying to get a "proper" world but haven't managed, the only thing that generates are the saved chunks and everything is bordered by cliffs into the void, which kind of defeats the purpose as the world is not playable.

**Comment by zekeflynn** [comment]
![image](https://user-images.githubusercontent.com/33818126/201001066-f5695449-9a08-442f-85f5-e429b9549b48.png)

I've even managed to "graft" it into another world by copying regions but the "void" regions seem locked to the downloaded world 😥no good
